openpype/plugins/publish/integrate_legacy.py

```diff
@@ -612,8 +612,14 @@ def register(self, instance, repres):
                     continue
                 repre_context[key] = template_data[key]
 
+            # Initialize ID, trying to get it from an earlier stage
```

Comment:
Conversation related to being able predefine representation id https://github.com/pypeclub/OpenPype/discussions/3902#discussioncomment-3794454

Reply:
This change is not needed anymore right?

Reply:
I'm currently developing the correct behaviour. It'll be put in new PR on this branch in some days/weeks. I'll notify you at this moment and it'll remove this change.